jupyter_scheduler/models.py

```diff
@@ -160,24 +160,46 @@ class DeleteJob(BaseModel):
     job_id: str
 
 
-class CreateJobDefinition(CreateJob):
+class CreateJobDefinition(BaseModel):
+    input_uri: str
+    output_prefix: str
+    runtime_environment_name: str
+    runtime_environment_parameters: Optional[Dict[str, EnvironmentParameterValues]]
+    output_formats: Optional[List[str]] = None
+    idempotency_token: Optional[str] = None
+    parameters: Optional[Dict[str, ParameterValues]] = None
+    tags: Optional[Tags] = None
+    name: Optional[str] = None
+    email_notifications: Optional[EmailNotifications] = None
```

Comment:
Let's remove email, timeout info, retry info from our APIs until we fully implement it. Can you open an issue to track that work and remove from the APIs in the meantime?

Reply:
Created #90.